sdk/spring/spring-integration-azure-eventhubs/src/main/java/com/azure/spring/integration/eventhubs/inbound/health/EventHusProcessorInstrumentation.java

```diff
@@ -23,12 +23,25 @@
 
     private ErrorContext errorContext;
 
+    /**
+     * Construct a {@link EventHusProcessorInstrumentation} with the specified name,type and noneErrorWindow.
+     *
+     * @param name the name
+     * @param type the type
+     * @param noneErrorWindow the none error window
```

Comment:
how about introducint the noneErrorWindow as "the period of a none error window"? and same for the consturctor